galley/pkg/crd/validation/validation.go

```diff
@@ -0,0 +1,139 @@
+
+
+
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package validation
+
+import (
```

Comment:
FYI, you'll need to run `make format` and `make lint` to help fix the formatting and linter issues before the PR can be merged.

Reply:
sure, Thanks~